core/state/iterator.go

```diff
@@ -109,7 +109,7 @@ func (it *NodeIterator) step() error {
 	if err := rlp.Decode(bytes.NewReader(it.stateIt.LeafBlob()), &account); err != nil {
 		return err
 	}
-	dataTrie, err := it.state.db.OpenStorageTrie(it.state.originalRoot, common.BytesToHash(it.stateIt.LeafKey()), account.Root)
+	dataTrie, err := it.state.db.OpenStorageTrie(it.state.originalRoot, common.Address{} /* FIXIT */, account.Root)
```

Comment:
Note to self here: `NodeIterator` is only used in tests, so we could enable preimage in these tests

Reply:
fixed